### ISSUE TITLE
t/writeline.t fails (with older Test2::Bundle::Extended?)

### ISSUE THREAD

**Issue by eserte** [CpanelInc/Test-MockFile#28]
On some of my smokers t/writeline.t fails. A sample report: http://www.cpantesters.org/cpan/report/b6c87bde-0ecd-11e9-b0ff-b8c4ea7c92bb

Statistical analysis suggests that this happens with older Test2::Bundle::Extended:
```
****************************************************************                                                                                                                                                                                                          
Regression 'mod:Test2::Bundle::Extended'                                                                                                                                                                                                                                  
****************************************************************                                                                                                                                                                                                          
Name                   Theta          StdErr     T-stat                                                                                                                                                                                                                   
[0='const']          -0.0000          0.0000      -3.38                                                                                                                                                                                                                   
[1='eq_0.000032']            -0.0000          0.0000      -1.60                                                                                                                                                                                                           
[2='eq_0.000058']             1.0000          0.0000    8791987084298526.00                                                                                                                                                                                               
[3='eq_0.000060']             1.0000          0.0000    5755706617553155.00                                                                                                                                                                                               
[4='eq_0.000067']             1.0000          0.0000    8791987084298528.00                                                                                                                                                                                               
[5='eq_0.000070']             1.0000          0.0000    9631139302728756.00                                                                                                                                                                                               
[6='eq_0.000072']             1.0000          0.0000    11511413235106310.00                                                                                                                                                                                              
[7='eq_0.000077']             1.0000          0.0000    9631139302728754.00                                                                                                                                                                                               
[8='eq_0.000083']             1.0000          0.0000    5755706617553157.00                                                                                                                                                                                               
[9='eq_0.000097']             1.0000          0.0000    5755706617553153.00                                                                                                                                                                                               
[10='eq_0.000100']            1.0000          0.0000    5755706617553157.00                                                                                                                                                                                               
[11='eq_0.000108']            1.0000          0.0000    5755706617553156.00                                                                                                                                                                                               
[12='eq_0.000111']            1.0000          0.0000    7614084164747197.00                                                                                                                                                                                               
[13='eq_0.000114']            1.0000          0.0000    11795688466675010.00                                                                                                                                                                                              
[14='eq_0.000115']            1.0000          0.0000    13429982107624026.00                                                                                                                                                                                              
[15='eq_0.000116']            1.0000          0.0000    12870151255394236.00                                                                                                                                                                                              
[16='eq_0.000117']            1.0000          0.0000    14285288145180914.00                                                                                                                                                                                              
                                                                                                                                                                                                                                                                          
R^2= 1.000, N= 125, K= 17                                                                                                                                                                                                                                                 
****************************************************************                                                                                                                                                                                                          
```
So it fails if 0.000032 is installed.

**Comment by toddr** [comment]
The failing code is 
```perl
isa_ok( $fh, ["IO::File"], '$fh is a IO::File' );
```

Test names for isa_ok weren't introduced until 0.000035 per https://metacpan.org/changes/release/EXODIST/Test2-Suite-0.000084

**Comment by toddr** [comment]
As best I can tell, this is fixed in 5473e307d9049b39a9f7ea9f3a1dcc53d3aac0a7
